rag/sql_rag/r2r_vector_store.py

```
@@ -0,0 +1,318 @@
+                records.append({"id": doc["id"], "training_data_type": "sql", "question": doc["metadata"]["question"], "content": doc["metadata"]["sql"]})
+        
+        return pd.DataFrame(records)
+
+    def remove_training_data(self, id: str, **kwargs) -> bool:
+        response = self._run_request("DELETE", f"collections/{self.collection_name}/documents/{id}")
+        return response is not None
+
+
+
+class IOIntelligence(VannaBase):
+  def __init__(self, config={}):
```

Ruff (0.12.2)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)